Jira Add Authentication (#15618)
demisto/content: Packs/Jira/Integrations/JiraV2/JiraV2_description.md

Fetching incidents by creation time (using the Created field), instead of using IDs, is done by checking the "Use created field to fetch incidents" checkbox.

You can authenticate using one of the following methods:

##### 1. Basic Authentication:
As of June 2019, Basic authentication with passwords for Jira is no longer supported, please use basic authentication using API Token or use OAuth 1.0. To use basic authentication, follow [this tutorial](https://confluence.atlassian.com/cloud/api-tokens-938839638.html) to get the API token.

##### 2. OAuth1.0:
To use OAuth1.0 follow [this tutorial](https://developer.atlassian.com/cloud/jira/platform/jira-rest-api-oauth-authentication/) to get the Access Token. Authorizing using OAuth1.0, requires Access Token, Private Key, and Consumer Key. 

##### 3. Private Access Token

> To use Private Access Token, follow [this tutorial](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) to get Access Tokens. Authorizing using PAT requires Access Token Only.   

Reviewer: ```suggestion
To use Private Access Token, follow [this tutorial](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) to get Access Tokens. Authorizing using PAT requires Access Token only.
Insert the generated token in the Access Token field.
```

Fetching incidents by creation time (using the Created field), instead of using IDs, is done by checking the "Use created field to fetch incidents" checkbox.

You can authenticate using one of the following methods:

##### 1. Basic Authentication:
As of June 2019, Basic authentication with passwords for Jira is no longer supported, please use basic authentication using API Token or use OAuth 1.0. To use basic authentication, follow [this tutorial](https://confluence.atlassian.com/cloud/api-tokens-938839638.html) to get the API token.

##### 2. OAuth1.0:
To use OAuth1.0 follow [this tutorial](https://developer.atlassian.com/cloud/jira/platform/jira-rest-api-oauth-authentication/) to get the Access Token. Authorizing using OAuth1.0, requires Access Token, Private Key, and Consumer Key. 

##### 3. Private Access Token

> To use Private Access Token, follow [this tutorial](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) to get Access Tokens. Authorizing using PAT requires Access Token only.

Reviewer 2: ```suggestion
To use a Private Access Token, please follow [this tutorial](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) to obtain your Access Tokens. Authorizing using a PAT requires the Access Token only.
```

> Insert the generated token in the Access Token field.

Reviewer 2: ```suggestion
Insert the generated token in the `Access Token` field.
```
